### DISCUSSION TITLE
How to Point to a Differently Named Default Values File in HelmRelease

### DISCUSSION THREAD

**Discussion by SachithKasthuriarachchi** [fluxcd/flux2#5053] [Q&A] [answered]
I have a helm chart which has default values file in following names based on the cloud provider I install them on.

- azure.yaml
- aws.yaml
- gcp.yaml

How to ask flux to consider the 'azure.yaml' as the default values file when creating a HelmRelease resource in azure. (this azure.yaml file is available within the helmchart. it is not something I am customizing)

**Comment by stefanprodan** [comment, marked as answer]
See https://fluxcd.io/flux/components/source/helmcharts/#values-files

**Reply by SachithKasthuriarachchi** [reply]
oh thanks a lot. I searched this in the docs but couldn't find it. thanks again.
